### PR TITLE
Fix #883, #884: Add post-install message and survey links

### LINKED ISSUE #883: cli: add github,survey links to help message
add a splash screen at a random that invites the users to fill out a survey, add a star on github, join slack, etc. This will also include the option so users specify they don't want to see the splash screen at all in the future.

### PR DESCRIPTION
**Pull request by rishiUT**
Created messages for users of Popper fulfilling the requirements laid out in issues #883 and #884

### CODE REVIEW

**Comment by rishiUT** [comment]
Hi! Is there anything we should do to improve this pull request?

File: install.sh [hunk 1]
```diff
@@ -22,6 +22,12 @@ install_system_wide() {
   fi
   echo
   echo "Popper is now available for all users in this system!"
+  echo "You can discuss Popper in our slack using the following link:"
+  echo "https://bit.ly/join-popper-slack"
+  echo "Also, please give us your feedback using this survey! https://bit.ly/popper-survey"
```
Comment (ivotron) [inline install.sh:27]: can we make this say "... your feedback via this 2-min survey ..."

**Review by ivotron — COMMENTED** [review]
sorry for the delay @rishiUT, a minor comment and it should be ready to be merged!

**Review by ivotron — APPROVED** [review]

**Comment by ivotron** [comment]
thanks a lot @rishiUT!

**Pull request merged**